SDL/Constants.h

```diff
@@ -52,16 +52,11 @@ typedef float FPX_seg;
 
 const unsigned int MAX_BLOCKS = 80;
 const unsigned int MAX_CONNECTED_MODULES = 40;
-const unsigned int N_MAX_MD_PER_MODULES = 89;
-const unsigned int N_MAX_SEGMENTS_PER_MODULE = 537; //Also used in (pixel)trackletDefualtAlgo in header files-> change there as well (hard coded)
 const unsigned int N_MAX_PIXEL_MD_PER_MODULES = 100000;
```

Comment:
Are these still needed?

Reply:
No, these two are not used anymore

Reply:
I am not talking about the ones you deleted. The comment is on the next line.

Is `N_MAX_PIXEL_MD_PER_MODULES` and `N_MAX_PIXEL_SEGMENTS_PER_MODULE` still needed?

Reply:
the N_MAX_Pixel is still needed, because memory related to pixel is not optimized and still used as static allocation